logs/parse.go

```diff
@@ -113,13 +113,9 @@ func IsSupportedPrefix(prefix string) bool {
 	return false
 }
 
-func ParseLogLineWithPrefix(prefix string, line string) (logLine state.LogLine, ok bool) {
+func ParseLogLineWithPrefix(prefix string, line string, tz *time.Location) (logLine state.LogLine, ok bool) {
 	var timePart, userPart, dbPart, appPart, pidPart, logLineNumberPart, levelPart, contentPart string
 
-	// Assume Postgres time format unless overriden by the prefix (e.g. syslog)
-	timeFormat := "2006-01-02 15:04:05 -0700"
-	timeFormatAlt := "2006-01-02 15:04:05 MST"
```

Comment:
The general idea is these are no longer locals of this big function but pushed into the `getOccurredAt` helper.

Reply:
Yeah I was going to suggest this earlier too, I think it's good that you moved this to `getOccurredAt` 👍